bql/semantic/expression_test.go

```diff
@@ -15,11 +15,13 @@
 package semantic
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/google/badwolf/bql/lexer"
 	"github.com/google/badwolf/bql/table"
 	"github.com/google/badwolf/triple/literal"
+	"github.com/google/badwolf/triple/node"
 )
 
 func buildLiteral(l string) string {
```

Comment:
Did you check if there is any usage for this, now?
If not, could you remove it, please?

Reply:
The function `buildLiteral` is indeed not being used anymore, I have just removed it.